### PR TITLE
feature(redis): lua script words

### PR DESCRIPTION
**Pull request by rgrinberg**
Add words to run/load/debug lua scripts

### CODE REVIEW

File: extra/redis/command-writer/command-writer.factor [hunk 1]
```diff
@@ -23,6 +23,16 @@ M: sequence write-resp ( sequence -- )
     [ dup number? [ number>string ] when ] map
     write-resp ;
 
+: write-command-multi ( sequence command -- )
+    swap append
```
Comment (mrjbq7) [inline extra/redis/command-writer/command-writer.factor:27]: prepend?

File: extra/redis/redis-tests.factor [hunk 2]
```diff
@@ -66,3 +66,17 @@ IN: redis.tests
         [ "hello" redis-incr ] [ drop t ] recover
     ] with-redis-test
 ] unit-test
+
+{ "e0e1f9fabfc9d4800c877a703b823ac0578ff8db" } [
+    [ "return 1" redis-script-load ] with-redis-test
+] unit-test
+
+{ { 0 0 } } [
+    [ { "foo" "bar" } redis-script-exists ] with-redis-test
```
Comment (mrjbq7) [inline extra/redis/redis-tests.factor:75]: should this return ``t/f`` instead of ``1/0``?

File: extra/redis/command-writer/command-writer.factor [hunk 3]
```diff
@@ -122,3 +132,13 @@ PRIVATE>
 ! Remote server control
 : info ( -- ) { "INFO" } write-resp ;
 : monitor ( -- ) { "MONITOR" } write-resp ;
+
+! Lua
+: script-load ( script -- ) 1array { "SCRIPT" "LOAD" } write-command-multi ;
+: script-exists ( scripts -- ) { "SCRIPT" "EXISTS" } write-command-multi ;
+: script-flush ( -- ) { } { "SCRIPT" "FLUSH" } write-command-multi ;
+: script-kill ( -- ) { } { "SCRIPT" "KILL" } write-command-multi ;
+! YES | SYNC | NO
```
Comment (mrjbq7) [inline extra/redis/command-writer/command-writer.factor:141]: what does this comment mean?

**Comment by mrjbq7** [comment]
Cool! Made a few comments.

File: extra/redis/command-writer/command-writer.factor (hunk 3 above)
Comment (rgrinberg) [reply extra/redis/command-writer/command-writer.factor:141]: those are the possible arguments to the command. I.e. `SCRIPT DEBUG <YES|SYNC|NO>`

**Comment by mrjbq7** [comment]
Merged!

File: extra/redis/redis-tests.factor (hunk 2 above)
Comment (rgrinberg) [reply extra/redis/redis-tests.factor:75]: It can be. But I don't see any other commands massaging the redis output. Is it worth doing it here?

**Pull request merged**